src/daemon/types_list.c

```diff
@@ -104,7 +104,7 @@ static void parse_line(char *buf) {
   if (fields[0][0] == '#')
     return;
 
-  data_set_t ds = {{0}};
+  data_set_t ds = {{0}, 0, NULL};
```

Comment:
Why is this needed? When a struct is only partially initialized, the remainder is initialized to zero according to the C standard.

Reply:
`-Wextra` implies `-Wmissing-field-initializers` => I'll explicitly disable that from the tests, and drop this change.